docs/questions-and-answers.md

```diff
@@ -10,10 +12,76 @@ An Agent Service is an Autonomous Service which is implemented as a multi-agent
+Under the hood the ACN is a DHT that keeps track of live agents mapping their crypto address to IP address. So agents can communicate with other agents without knowing their network location assuming they are online or offline but registered in the ACN.
+</details>
+
+<details><summary>What happens when agents are deployed now?</summary>
+Currently only the so called "island deployments" are being operated, which are services that run as one-off services, not anchored in the protocol, because the protocol is not live (more on that <a href="https://www.autonolas.network/blog/11">here</a>). Once the protocol is live, agents will be able to interact with it so they can monetize their work and connect to other services.
+</details>
+
+<details><summary>How many composition levels does open autonomy offer?</summary>
+Composition starts at the component level of the agents (multiple rounds make a skill), then continues on agent level (multiple skills make an agent) and ends at service level (multiple agents make a service).
 </details>
+
+<details><summary>How do agents settle a transaction?</summary>
+
+<ol>
+<li>Negotiation happens through the ACN or alternatively another connection like the ABCI connection.</li>
+<li> A threshold of agents agree on a transaction hash.</li>
+<li>One of the agents is randomly selected as the keeper using a deterministic function based on a public, verifiable randomness source (currently DRAND).</li>
+<li>All agents sign the transaction using a multi-sig like Gnosis Safe.</li>
+<li>The keeper collects all the signatures and sends the transaction onchain.</li>
+<li>All agents wait for the transaction to be mined and validate the output.</li>
+<li>Done</li>
+</ol>
+</details>
+
+## Security
+
+<details><summary>How are Autonolas services run?</summary>
+
+Autonolas services are composed of multiple agents that run the same code and agree on its output. These agents are executed in different nodes that are run by independent operators. In order to avoid centralizing this power, Autonolas does not play a role here so it does not offer a cloud platform. Each operator must select and setup the infrastructure that best suits their needs.</details>
+
+<details><summary>What happens if my node is hacked?</summary>
+
+As in any other online service, Autonolas nodes are exposed to the risk of being breached. At the individual level, Autonolas does not provide a solution to this and it’s up to the node operator to keep the node safe. At the service level, on the other hand, services are secured in three ways:</p>
+
+<ul>
+<li>Each agent service implements a custom protocol that expects very narrow message flow, so a hypothetical node running malicious code would need to express its intentions within this protocol, otherwise the other agents will ignore its messages.</li>
```

Comment:
```suggestion
<li>Each agent service implements a custom protocol that expects very narrow message flow, so a hypothetical agent running malicious code would need to express its intentions within this protocol, otherwise the other agents will ignore its messages.</li>
```